job_templates/sag_pt_model_learner/config_fed_server.json

```diff
@@ -0,0 +1,76 @@
+{
+    "format_version": 2,
+    "server": {
+        "heart_beat_timeout": 600
+    },
+    "task_data_filters": [],
+    "task_result_filters": [],
+    "components": [
+        {
+            "id": "persistor",
+            "name": "PTFileModelPersistor",
+            "args": {
+                "model": {
+                    "path": "fl.net.Net",
```

Comment:
Suggest you use file format ".conf" (please refer to "sag_pt")
And add some comments here to explain that users might need to change this if they are using different network / model